src/types/datetime.rs

```diff
@@ -524,4 +566,33 @@ mod tests {
             assert!(b.unwrap().get_fold());
         });
     }
+
+    #[cfg(not(PyPy))]
```

Comment:
We should probably test this on PyPy too. Probably best to do that by adding test to `pytests/src/datetime.rs` (and corresponding python file).